runtime/v2/runc/v1/service.go

```diff
@@ -326,11 +326,13 @@ func (s *service) Exec(ctx context.Context, r *taskAPI.ExecProcessRequest) (*pty
 	if err != nil {
 		return nil, err
 	}
-	if container.ProcessExists(r.ExecID) {
+	ok, cancel := container.ReserveProcess(r.ExecID)
```

Comment:
This should be called `ok, release`

Reply:
cancel is pretty standard because of `context.WithCancel`

Reply:
I know but `cancel` is because of `context cancellation`. This isn't a context so I feel its misleading. For example `cancel` doesn't mean "roll back". `cancel()` doesn't always cancel, sometimes its already committed and `cancel()` does nothing. In general `release` seems to be more clear but I don't care that much.

Reply:
In the sense of "cancel the reservation" it makes sense to me; and since this is a nit, let's go with it. We can always update later if we want to use a different term.

Reply:
Yep np